.devcontainer/ubuntu-gui-full/devcontainer.json

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2024 Antmicro <www.antmicro.com>
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  "image": "ghcr.io/antmicro/openroad-flow-scripts:lxqt",
```

Comment:
I see other references to orfs:latest but this one was not updated.  Is that an oversight?

Reply:
Hey @maliberty,

I left this reference for testing purposes, intending to replace it with `http://ghcr.io/The-OpenROAD-Project/openroad-flow-scripts:lxqt` once the changes are reviewed. However, I understand there might be some confusion.

To clarify, the `http://ghcr.io/antmicro/openroad-flow-scripts:lxqt` image includes additional dependencies not present in `openroad/ORFS:latest`, so it cannot be replaced directly. If you prefer, I can update the reference to `openroad/ORFS:lxqt` instead.

Please let me know how you would like to proceed.

Reply:
What additional dependencies?

Reply:
The [Dockerfile](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1927/files#diff-0f739075066edddc833d91d8f4b547672a24ffb437a70b4785fda70c4ddbd384) consists of a couple of additional layers to the [ORFS Docker image](https://hub.docker.com/r/openroad/orfs). Those additions are targeted to create a desktop environment using X11, LXQT, and VNC for OpenROAD's GUI support through VNC in Codespaces.

To sum up:
  1. [The first layer](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1927/files#diff-0f739075066edddc833d91d8f4b547672a24ffb437a70b4785fda70c4ddbd384) is responsible for `X11`, `LXQT`, and other desktop environment dependencies installation.
  2. [The second layer](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1927/files#diff-0f739075066edddc833d91d8f4b547672a24ffb437a70b4785fda70c4ddbd384) is responsible for `noVNC` installation and configuration, so it can be used in Codespaces (e.g. starting `tigervnc` server, changing remote scaling, etc...).
  3. [The third layer](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1927/files#diff-0f739075066edddc833d91d8f4b547672a24ffb437a70b4785fda70c4ddbd384) adjusts LXQT theme
  4. [The fourth layer](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1927/files#diff-0f739075066edddc833d91d8f4b547672a24ffb437a70b4785fda70c4ddbd384) is responsible for VNC configuration, more precisely the scaling setting and skip of the login panel.